app/assets/stylesheets/application.scss

```diff
@@ -11,3 +11,4 @@
 // Your CSS partials
 @import "components/index";
 @import "pages/index";
+@import "components/card";
```

Comment:
should @import "card" in the component/index.scss file instead of importing in application.scss